src/_colors.css

```diff
@@ -0,0 +1,39 @@
+:root {
```

Comment:
It looks like some of these map to our brand colors, but some don't. For example, `blue-ocean` has the same values as `blue-slate`. Is that intentional? Also, I don't see `#f7f7f7` which is the white-offset hex code we use for bellycard smb.

For reference: https://github.com/bellycard/rolodex/blob/master/assets/stylesheets/rolodex/settings/variables/_colors.scss